modules/bbp-simulation/src/main/resources/schemas/hbphackathon/simulation/morphologyrelease/v0.1.0.json

```diff
@@ -35,7 +35,8 @@
               "path": "nsg:morphologyIndex",
               "name": "Morphology release index file",
               "description": "Morphology release index file.",
-              "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
+              "seeAlso": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/EntityShape",
+              "class": "prov:Entity"
             }
           ]
         }
```

Comment:
Should line 61 ("application/swc,application/neurolucida,application/h5,application/neuroml") be the array of strings?

Reply:
YEs.